pkg/shared/shared.go

```diff
@@ -47,3 +48,21 @@ func fileExists(p string) (bool, error) {
 	}
 	return false, fmt.Errorf("checking existance of file %s: %w", p, err)
 }
+
+// ServerAuthSecret returns the secret of the manage server using the secret
+// file as given in environment variable. In case of development it uses the
+// development password.
+func ServerAuthSecret(secretFile string, devEnv string) ([]byte, error) {
+	pw := []byte(DevelopmentPassword)
+	dev, _ := strconv.ParseBool(devEnv)
+	// Error value does not matter here. In case of an error dev is false and
+	// this is the expected behavior.
+	if !dev {
```

Comment:
In go, you should try not to indent the happy path: https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88

I would do it like this:

```go
func ServerAuthSecret(secretFile string, devEnv string) ([]byte, error) {
  if dev, _ := strconv.ParseBool(devEnv); dev {
    return []byte(DevelopmentPassword), nil
  }

  pw, err := os.ReadFile(...)
  ...
  return pw, nil
}
```